kibbeh/src/ui/NativeRadio.tsx

```diff
@@ -1,116 +1,90 @@
-/* eslint-disable operator-linebreak */
-import { Console } from "node:console";
-import React, { ChangeEvent, HTMLAttributes, ReactNode } from "react";
-import SolidMoon from "../icons/SolidMoon";
+import React, { useState } from "react";
 
 export interface NativeRadioProps {
-  icon?: string | React.SVGProps<SVGSVGElement>;
+  icon?: React.ReactElement;
   title: string;
-  description: string;
-  id?: Settings;
+  subtitle: string;
   checked?: boolean;
-  radios?: Array<RadioOptions>;
-  children?: ReactNode;
+  onClick?: (id: number | undefined) => void;
+  num?: number;
 }
 
-export type RadioOptions = Omit<NativeRadioProps, "children">;
-
-export type Settings = "online" | "not-disturb" | "dark-theme" | "light-theme";
-
-const optionalChangedColor = "text-primary-300";
-
-export function NativeRadioList({
-  radios,
+export const NativeRadio: React.FC<NativeRadioProps> = ({
   icon,
   title,
-  id,
+  subtitle,
   checked = false,
-  description,
-  children,
-  ...args
-}: NativeRadioProps) {
-  const [radioSelected, setRadioSelected] = React.useState<
-    NativeRadioProps | boolean
-  >(false);
-  const radioRef = React.useRef<HTMLInputElement>(null);
-
-  const setSetttings = (e: React.MouseEvent) => {
-    const { current } = radioRef;
-    if (current && !current.contains(e.target as any)) {
-      setRadioSelected(false);
-    }
-
-    console.log(radioRef.current?.value);
-  };
-
+  onClick,
+  num,
+}) => {
   return (
-    <React.Fragment>
-      <div
-        style={{
-          width: "601px",
-          height: "64px",
-          borderRadius: "8px",
-          background: "#151A21",
-        }}
-        className="flex justify-between mt-4"
-      >
-        <div
-          style={{ marginTop: "16px", marginBottom: "38px" }}
-          className={icon ? `flex flex-none absolute ml-3 mr-3` : `absolute `}
-        >
-          {icon}
-        </div>
-        <div className={icon ? `ml-6 mt-2 mb-2 relative` : `ml-3 mt-2 mb-2`}>
-          <span
-            className={
-              radioSelected !== checked
-                ? `text-primary-100 font-bold`
-                : `${optionalChangedColor} font-bold`
-            }
+    <button
+      className="w-full flex px-3 py-2 bg-primary-900 rounded-8 justify-between group"
+      onClick={onClick ? () => onClick(num) : undefined}
+    >
```

Comment:
On each card, there was a noticeable white outline that's not visibly pleasing. Not sure if you want to remove it or keep it. `focus:outline-none` would help

Reply:
Yeah, Ben wants to keep that